src/validation.cpp

```diff
@@ -1398,7 +1398,7 @@ void CChainState::InvalidBlockFound(CBlockIndex* pindex, const BlockValidationSt
         if (VeriBlock::isCrossedBootstrapBlock()) {
             VeriBlock::GetPop()
                     .getAltBlockTree()
-                    .invalidateSubtree(pindex->GetBlockHash().asVector(), altintegration::BLOCK_FAILED_BLOCK);
+                    .invalidateSubtree(pindex->GetBlockHash().getReversed().asVector() , altintegration::BLOCK_FAILED_BLOCK);
```

Comment:
You can write now: `pindex->GetBlockHash().asReversed()`
Also remove extra space.